pkg/client/connector/command.go

```diff
@@ -218,11 +219,11 @@ func (s *service) connectWorker(c context.Context, cr *rpc.ConnectRequest, k8sCo
 	connectStart := time.Now()
 
 	dlog.Info(c, "Connecting to traffic manager...")
-	tmgr, err := newTrafficManager(c,
+	tmgr, err := worker_trafficmanager.NewTrafficManager(c,
```

Comment:
The naming of the package and the constructor seems a bit off. Instead of repeated names in package and constructor, how about `trafficmanager.NewWorker()` ? If we want to have `worker` in the package structure, then perhaps use `pkg/client/connector/worker/trafficmanager`, `pkg/client/connector/worker/grpc`, and `client/connector/worker/cluster`.